src/phoenix/datasets/types.py

```diff
@@ -183,3 +208,196 @@ def __post_init__(self) -> None:
     Callable[[Example], TaskOutput],
     Callable[[Example], Awaitable[TaskOutput]],
 ]
+
+
+@dataclass(frozen=True)
+class ExperimentConfig:
+    n_examples: int
```

Comment:
is this really a config of an experiment? It's the cardinality of the dataset examples no?

Reply:
kind of, since the size of the dataset can grow or shrink and each experiment is just a snapshot

Reply:
renamed to `Params`